deepchem/models/losses.py

```diff
@@ -205,6 +205,114 @@ def loss(output, labels):
     return loss
 
 
+class VAE_ELBO(Loss):
+  """The Variational AutoEncoder loss, KL Divergence Regularize + marginal log-likelihood.
+  
+  The logvar and mu should have shape (batch_size, hidden_space).
```

Comment:
Could you add a references section to this docstring with links to papers introducing the ELBO loss for VAEs? Check out the numpy docs on how to add reference sections to docstring: https://numpydoc.readthedocs.io/en/latest/format.html#id8

Reply:
Could you also add an Examples section showing how to invoke the loss function?

Reply:
Added completed. But, I'm not sure this is a good example.